test/Syntax/self_cannot_be_found.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-typecheck-verify-swift
+
+let _ = self // expected-error {{cannot find `self` in scope, did you mean to declare or extend a type?}} 
+let s = self.b // // expected-error {{cannot find `self` in scope, did you mean to declare or extend a type?}} 
+let w = self.method() // expected-error {{cannot find `self` in scope, did you mean to declare or extend a type?}} 
```

Comment:
Oh actually these will need updating for the new diagnostic text

Reply:
Oh you’re right, just a second

Reply:
There